### DISCUSSION TITLE
Welcome to CoquiArts Discussions!

### DISCUSSION THREAD

**Discussion by BraianPerez97** [BraianPerez97/CoquiArts#1] [Announcements]
### Hello, fellow humans. Welcome to CoquiArts "If it's Art, check out CoquiArts"

_The first part of our project is the research._

**We will send you an invite link to a google drive containing our research.**

- We will be reading 1 module a week to share notes on PLD and share our notes.

- After PLD's are done in October, we will still be meeting every monday to discuss and share our notes.

- On the research phase, since we are being monitored, person that misses a total of 3 meetings will be left out of the group. Meetings can be considered being hosted on google meet, although we rather and should have them in person.

- The module we have to read is SQL and ORM, we will share our notes the upcoming 1 of September. Be Prepared.

- Following this we will have 1 week to research our next topic and prepare to share said notes on PLD. 

- Please do the projects of said modules to further expand and understand the technology and use-cases.

- Next two modules to be discussed is Express.js and Node.js due the 8 of September. Again be Prepared.

- Express.js

**Welcome to the book club my fellow humans!**

**---It is our responsibility to be prepared.---
----- PLEASE READ THEM AND INTERNALIZE IT -----
WE WILL BE DISCUSSING THIS IN OUR WEEKLY MEETINGS!**

**Comment by nashalysf** [comment]
![Coqui Arts Project Timeline Graph](https://github.com/BraianPerez97/CoquiArts/assets/107204057/e80fb503-4ef7-4da4-93fc-99f5ade60518)
### Remember guys:
During the production of the app, we still have to create presentations, profiles and careers for Holberton,
#### Don't let work accumulate!

**Reply by GuillermoRosario** [reply]
Vamos pa encimaaa
